### PR TITLE
documentation patch

### PR DESCRIPTION
**Pull request by sturkmen72**
https://github.com/Itseez/opencv/pull/5348

### CODE REVIEW

File: modules/imgcodecs/include/opencv2/imgcodecs.hpp [hunk 1]
```diff
@@ -68,12 +68,12 @@ enum ImreadModes {
...
        IMREAD_LOAD_GDAL            = 8,  //!< If set, use the gdal driver for loading the image.
-       IMREAD_GRAYSCALE_REDUCED_2  = 16, //!< If set, always convert image to the single channel grayscale image and the image size reduced 1/2.
-       IMREAD_COLOR_REDUCED_2      = 17, //!< If set, always convert image to the 3 channel BGR color image and the image size reduced 1/2.
-       IMREAD_GRAYSCALE_REDUCED_4  = 32, //!< If set, always convert image to the single channel grayscale image and the image size reduced 1/4.
-       IMREAD_COLOR_REDUCED_4      = 33, //!< If set, always convert image to the 3 channel BGR color image and the image size reduced 1/4.
-       IMREAD_GRAYSCALE_REDUCED_8  = 64, //!< If set, always convert image to the single channel grayscale image and the image size reduced 1/8.
-       IMREAD_COLOR_REDUCED_8      = 65  //!< If set, always convert image to the 3 channel BGR color image and the image size reduced 1/8.
+       IMREAD_REDUCED_GRAYSCALE_2  = 16, //!< If set, always convert image to the single channel grayscale image and the image size reduced 1/2.
```
Comment (sturkmen72) [inline modules/imgcodecs/include/opencv2/imgcodecs.hpp]: i wanted to rename newly added enum names but this causes error on building. how can i solve it ? @vpisarev , @mshabunin could you help me.

**Comment by alalek** [comment]
You must rename it everywhere, include implementation part, not declarations only.

**Comment by sturkmen72** [comment]
oh, sorry i forgot that i used them on the source. thank you.

**Comment by mshabunin** [comment]
:+1:

**Pull request merged**